examples/upstream_nameservers/main.tf

```diff
@@ -23,11 +23,6 @@ provider "google" {
   region  = var.region
 }
 
-provider "google-beta" {
```

Comment:
Why are we removing provider definition here and updating with stable in the [other example](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/228/files#diff-f5c29161b97122fb4fcac300207057afR17)

Reply:
> Why are we removing provider definition here and updating with stable in the [other example](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/228/files#diff-f5c29161b97122fb4fcac300207057afR17)

this one is still valid, did you want to remove provider or change to stable here?